tools/emhermesc/test.js

```diff
@@ -46,3 +49,5 @@ badBuf = goodBuf.slice(0, 140);
 assert.equal(wrap(() => hc.validateBytecodeModule(badBuf, 0)), "bytecode buffer is too small");
 
 assert.notEqual(wrap(() => hc.validateBytecodeModule(goodBuf, 0)), 0);
+
+console.log("test.js: success");
```

Comment:
Is having an extra log useful here?
Seems like we already got a `Success!` at the end?